bigbluebutton-html5/imports/startup/client/auth.js

```diff
@@ -3,16 +3,20 @@ import { setCustomLogoUrl } from '/imports/ui/components/user-list/service';
 import { log, makeCall } from '/imports/ui/services/api';
 import deviceInfo from '/imports/utils/deviceInfo';
 import logger from '/imports/startup/client/logger';
+import { Session } from 'meteor/session';
 
 // disconnected and trying to open a new connection
 const STATUS_CONNECTING = 'connecting';
 
-export function joinRouteHandler(nextState, replace, callback) {
-  const { sessionToken } = nextState.location.query;
+export function joinRouteHandler(callback) {
+  const urlParams = new URLSearchParams(window.location.search);
+  const sessionToken = urlParams.get('sessionToken');
+  console.log('joinRouteHandler_2', sessionToken);
 
-  if (!nextState || !sessionToken) {
-    replace({ pathname: '/error/404' });
-    callback();
+  if (!sessionToken) {
+    Session.set('hasError', true);
+    Session.set('codeError', '404');
+    callback('failed - no sessionToken');
```

Comment:
the expected params are `(value, error)` you probably should change the callback call or the parameters to reflect the return.